fhi_users/models.py

```diff
@@ -39,89 +38,62 @@ def get_highest_role(cls, is_patient, is_professional, is_admin):
             return cls.NONE
 
 
-# Auth-ish-related models
 class UserDomain(models.Model):
+    """
+    Domain model representing a user domain.
+    
+    The 'internal_phone_number' field is not unique and caution should be taken when
+    using it for lookups.
+    """
     id = models.CharField(
         max_length=300,
         primary_key=True,
         default=uuid.uuid4,
         editable=False,
         unique=True,
     )
-    # Money
     stripe_subscription_id = models.CharField(max_length=300, null=True)
-    # Info
-    # https://docs.djangoproject.com/en/5.1/ref/models/fields/#django.db.models.Field.null
     name = models.CharField(blank=True, null=True, max_length=300, unique=True)
     active = models.BooleanField()
-    # Business name can be blank, we'll use display name then.
     business_name = models.CharField(max_length=300, null=True)
     display_name = models.CharField(max_length=300, null=False)
     professionals = models.ManyToManyField("ProfessionalUser", through="ProfessionalDomainRelation")  # type: ignore
-    # The visible phone number should be unique... ish? Maybe?
-    # We _could_ allow users to log in with visible phone number IFF
-    # it's unique among active domains. We're going to TRY and have it
-    # be unique and hope we don't have to remove this. The real world is
-    # tricky.
     visible_phone_number = models.CharField(max_length=150, null=False, unique=True)
-    internal_phone_number = models.CharField(
-        max_length=150, null=True, unique=False, blank=True
-    )
+    internal_phone_number = models.CharField(max_length=150, null=True, blank=True)
     office_fax = models.CharField(max_length=150, null=True, blank=True)
     country = models.CharField(max_length=150, default="USA")
     state = models.CharField(max_length=50, null=False)
     city = models.CharField(max_length=150, null=False)
     address1 = models.CharField(max_length=200, null=False)
     address2 = models.CharField(max_length=200, null=True, blank=True)
     zipcode = models.CharField(max_length=20, null=False)
-    # Customize the defaults
-    default_procedure = models.CharField(
-        blank=False, null=True, max_length=300, unique=False
-    )
+    default_procedure = models.CharField(blank=False, null=True, max_length=300, unique=False)
     cover_template_string = models.CharField(max_length=5000, null=True)
 
-    def save(self, *args, **kwargs):
-        # Strip URL prefixes from name if it's set
-        if self.name:
-            self.name = self._clean_name(self.name)
-        super().save(*args, **kwargs)
-
```

Comment:
This seems bad where are we going to strip the URL prefix on making a new userdomain?